compiler/modules/bir/bir.bal

```diff
@@ -134,15 +134,20 @@ public type Register readonly & record {|
     int number;
     SemType semType;
     string? varName;
+    Position? pos;
 |};
 
-public function createRegister(FunctionCode code, SemType semType, string? varName = ()) returns Register {
+public function createRegister(FunctionCode code, SemType semType, string? varName = (), Position? pos = ()) returns Register {
     int number = code.registers.length();
-    Register r = { number, semType, varName };
+    Register r = { number, semType, varName, pos };
     code.registers.push(r);
     return r;
 }
 
+public function getRegister(FunctionCode code, int registerNo) returns Register {
```

Comment:
Please don't abbreviate `Number` to `No` (too much like `Yes`). Either don't abbreviate or abbreviate to `Num`.

Reply:
fixed